README.rst

```diff
@@ -19,19 +13,23 @@ the project so far.**
 
 This Napp allows a user to create a point to point L2 Ethernet Virtual Circuit.
 
-When fully implemented, this napp will provide a REST API to create/modify/delete circuits. For now, the list of installed circuits is kept in memory,
-but it should be later kept in a permanent storage.
-Circuits will be installed at the time of request or at a predefined time, and can also have a time to be deleted.
-The Napp also will listen for PortStatus events, modifying circuits that use a port that went down.
+Features
+========
+- REST API to create/modify/delete circuits;
+- REST API to create/modify/delete circuit scheduling;
+- list of circuits in memory and also synchronized to a permanent storage;
+- circuits can be installed at time of request or have an installation schedule;
+- circuits can use a predefined path or find a dynamic path;
+- the NApp will move circuits to another path in case of a link down;
+- web UI for circuits management.
 
-The cookie flow field is used to identify to which EVC the flow belongs. The first two bytes of the cookie is a prefix identifying the NApp using it,
-and the remaining 14 bytes are the EVC id.
 
 Requirements
 ============
 - kytos/flow_manager
 - kytos/pathfinder
 - kytos/topology
+- amlight/sndtrace_cp
 
 
 .. TAGs
```

Comment:
Maybe we can change from kytos to kytos-ng on the targets below

Reply:
I didn't change because we didn't rename the NApps, they are still called `kytos/<name>`.

Reply:
I mean the Build and Coverage tags (they are pointing to scrutinize's org "kytos", while should be kytos-ng). For the License, I think we should also use our kytos-ng image (https://img.shields.io/github/license/kytos-ng/kytos.svg) and target (https://github.com/kytos-ng/mef_eline/blob/master/LICENSE). Wdyt?

Reply:
Also related to this, @ajoaoff @italovalcy what you think of adding a `|Tag|`, like we have on `kytos`, `python-openflow`, and also recently on [`noviflow`](https://raw.githubusercontent.com/kytos-ng/noviflow/master/README.rst)? I believe that would also facilitate for operators to spot whichever latest tag is available.

Reply:
> Also related to this, @ajoaoff @italovalcy what you think of adding a `|Tag|`, like we have on `kytos`, `python-openflow`, and also recently on [`noviflow`](https://raw.githubusercontent.com/kytos-ng/noviflow/master/README.rst)? I believe that would also facilitate for operators to spot whichever latest tag is available, that way also encouraging the team to also push upstream the tag when newer versions are released

I believe it is a good idea!